popupDialogExample/PopupDialogExample.js

```diff
@@ -1,47 +1,65 @@
-import React, { Component } from 'react';
-import { View, Text, StyleSheet } from 'react-native';
-import { RoundButton } from 'react-native-button-component';
+import React, {Component} from 'react';
+import {View, Text, StyleSheet, TouchableHighlight, Dimensions} from 'react-native';
 import PopupDialog from 'react-native-popup-dialog';
 
+const {width, height} = Dimensions.get('window');
```

Comment:
then wouldn't be 
`const { WIDTH:width, HEIGHT:height } = Dimensions.get('window');` 
?